app/main.py

```diff
@@ -1,8 +1,36 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str) \
+            -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float, clean_power: int,
+                 average_rating: float, count_of_ratings: int) -> None:
```

Comment:
Place each argument in a [new line](https://google.github.io/styleguide/pyguide.html#3192-line-breaking), it will allow developers to quicker understand expected arguments and typing.